ocaml/typing/typecore.ml

```diff
@@ -6593,8 +6601,8 @@ and type_application env app_loc expected_mode pm
         end
       in
       if !Clflags.principal then begin_def () ;
-      let sargs = List.map 
-        (fun (label, e) -> Typetexp.transl_label label, e) sargs 
+      let sargs = List.map
+        (fun (label, e) -> Typetexp.transl_label label None, e) sargs
```

Comment:
Re: https://github.com/ocaml-flambda/flambda-backend/pull/1589#discussion_r1273874991
I think there is no hope of considering Position here, as mentioned in meeting we will have to check application for Positions the same way we consider Optional

Reply:
Let's look at this one together.

Reply:
Added comments for this